main/acle.md

```diff
@@ -2265,6 +2270,21 @@ is implementation-defined.
 The generic itanium C++ ABI, which we use in AArch64, already handles
 arrays with arbitrarily aligned elements
 
+## Scalable Vector Extension procedure call standard attribute
+
+On SVE enabled AArch64 targets, the [[AAPCS]](#AAPCS) allows for procedure
```

Comment:
Sorry, it is the AAPCS64 that we need to reference, for 64-bit targets. AAPCS is for the 32-bit one.

```suggestion
On SVE enabled AArch64 targets, the [[AAPCS64]](#AAPCS64) allows for procedure
```

unfortunately both references AAPCS and AAPCS64 point at the same link... that needs fixing, but in a separate patch.

Reply:
Done.